python/triton/compiler/compiler.py

```diff
@@ -617,7 +617,7 @@ def __getitem__(self, grid):
 
         def runner(*args, stream=None):
             if stream is None:
-                if self.device_type in ["cuda", "rocm"]:
+                if self.device_type in ["cuda", "hip"]:
```

Comment:
@micmelesse
I've found that we use `hip` in other places like this, so I consider this as a typo.
Do you mind if I create an upstream PR for this?

Reply:
Yup. The device_type should be "hip".

Background: The usage of "hip" instead of "rocm" stems from Pytorch which has "hip" devices:
https://pytorch.org/docs/stable/notes/hip.html#checking-for-hip